infrastructure/public-api.yaml

```diff
@@ -264,6 +264,57 @@ paths:
                 #set($context.responseOverride.header.Content-Type = "application/jwt")
                 $output.jwt
 
+  /credential/issue-test:
```

Comment:
Probably don't even need this since the lambda isn't going to exist outside of dev and we're not integration testing it yet?